walk-in-interview/src/main/webapp/strings.en.js

```diff
@@ -67,6 +67,11 @@ AppStrings['log-in'] = {
 AppStrings['business-log-in'] = {
   'account': 'Account (Email)',
   'password': 'Password',
+  'empty-email': 'Empty email input',
+  'invalid-email-error': 'The email address is not valid',
+  'user-disabled': 'The user corresponding to the given email has been disabled',
```

Comment:
what does disabled user mean?

Reply:
A disabled user cannot sign in to an app anymore and continues to be stored in the Firebase Auth backend.

Reply:
I think you should change the error message shown to the user so it's more clear what the error is. How does a user become disabled?